tests/analyze_ecl_failure.sh

```diff
@@ -24,7 +24,7 @@ do
     for line in $alines
     do
       abs_new=`echo $line| awk -F ' ' '{print $5}'`
-      abs_new=`echo ${abs_new: : -1} | awk '{printf sprintf("%.16f", $1); }'`
+      abs_new=`echo ${abs_new} | sed -e 's/\.$//g' | awk '{printf sprintf("%.16f", $1); }'`
```

Comment:
I do not think that this is the same. Currently you are stripping a dot at the end while the previous version is always stripping the last character.